ovirtlago/data/ovirt_cpu_map.yaml

```diff
@@ -8,12 +8,21 @@ Intel:
   Conroe: Intel Conroe Family
   Penryn: Intel Penryn Family
   Nehalem: Intel Nehalem Family
+  Nehalem-IBRS: Intel Nehalem-IBRS Family
   Westmere: Intel Westmere Family
+  Westmere-IBRS: Intel Westmere-IBRS Family
   SandyBridge: Intel SandyBridge Family
+  SandyBridge-IBRS: Intel SandyBridg-IBRS Family
```

Comment:
Missing 'e' in SandyBridge (also below in IvyBridge-IBRS)

Reply:
Thanks. Fixed.